static/app/components/nav/config.tsx

```diff
@@ -52,6 +52,7 @@ export function createNavConfig({organization}: {organization: Organization}): N
     label: t('Insights'),
     icon: <IconGraph />,
     feature: {features: 'insights-entry-points'},
+    analyticsKey: 'insights',
```

Comment:
Just in case you would expect more keys in the future here, you could also

```suggestion
    analytics: {
      key: '...',
      ...space for other properties
    }
```
or
```suggestion
    trackAnalytics: () => ... 
```

Reply:
That's a good callout! Can definitely expand to this kind of pattern if it is needed in the future.